UnitTests/PaymentsCoreTests/HTTP_Tests.swift

```diff
@@ -0,0 +1,139 @@
+import XCTest
+@testable import PaymentsCore
+@testable import TestShared
+
+// swiftlint:disable force_unwrapping implicitly_unwrapped_optional
+class HTTP_Tests: XCTestCase {
+    
+    // MARK: - Helper Properties
+    
+    var fakeRequest = FakeRequest()
+    var fakeURLRequest = FakeRequest().toURLRequest(environment: .sandbox)!
+    let successURLResponse = HTTPURLResponse(url: URL(string: "www.test.com")!, statusCode: 200, httpVersion: "https", headerFields: [:])!
+    
+    var config: CoreConfig!
+    var mockURLSession: MockURLSession!
+    var sut: HTTP!
+    
+    // MARK: - Test lifecycle
+    
+    override func setUp() {
+        super.setUp()
+        
+        config = CoreConfig(accessToken: "mockAccessToken", environment: .sandbox)
+        
+        mockURLSession = MockURLSession()
+        mockURLSession.cannedError = nil
+        mockURLSession.cannedURLResponse = nil
+        mockURLSession.cannedJSONData = nil
+        
+        sut = HTTP(
+            urlSession: mockURLSession,
+            coreConfig: config
+        )
```

Comment:
I think this was just moved over, but I believe we can single line this and have it be under our liter rule. I also wonder if it'd be clearer if we changed this property name to `mockHTTP` instead of `sut` to match what we use elsewhere in tests. Thoughts?

Reply:
I think using `sut` (subject under test) is a good practice for us. We do it in a handful of other test files (AnalyticsEventRequest_Tests, DeviceInspector_Tests, DataCollector_Tests, and our Button tests). I know on Android they use it heavily too.

It makes very clear which object the file is testing. The `sut` should never be a mock, too.